game-rules/index.md

```diff
@@ -0,0 +1,24 @@
+## What are the rules for putting a game on Game Jolt?
+
+When you click or tap "Add a Game" on your developer dashboard, the first thing you'll see are the rules for uploading a game to Game Jolt. The rules are there for a reason. You have to agree to them before you add a game, and if your game breaks the rules, it will be removed. Read the rules and abide by them. That way everyone stays happy and nobody gets hurt. That last bit was a joke. Someone always gets hurt. Just kidding. The rules are serious, though.
+
+### The rules:
+
+- Game Jolt is currently only accepting free games. There must be no requirement to pay in order to play your game. 
+- Don't upload games you didn't make. If you are not the developer of the game, you cannot add the game to the site. If you're a fan, encourage the developer to add the game themselves! 
+- Game Jolt is a place for indie developers, not publishers or portals. If you did not create the game yourself, do not upload it, even if you own it. Remember: This is Indie. 
+- Our name is "Game Jolt". As such, only actual games may be uploaded to the site. No applications, software, or tools are allowed at this time. We may accept these in the future or on a case by case basis, but for now, let's just keep it to games. 
+- No stripped-down demos or "lite" versions of commercial games are permitted. You must upload a totally free version of your game, and it must be as complete a version as is publicly available. 
+- We welcome and encourage you to add games that are incomplete or works-in-progress, as long as they are tagged as In Development. However, you should keep your game hidden until you have something to show. In Development games do not have to be playable, but they must be further along than the concept stage. Before you unhide the game, make sure you at least have some screenshots and an informative description. 
+- Do not add links to download or play your game on other sites unless you have already uploaded it to Game Jolt. You can't use your game page as a big ad for another site. 
```

Comment:
can we reword the last sentence? "big ad" sounds weird. Maybe something like "You may not use your Game Jolt game page as an advertising platform for another site"